bitnami/metallb/Chart.yaml

```
@@ -30,4 +30,4 @@ maintainers:
 name: metallb
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/metallb
-version: 4.5.7
+version: 4.5.8
```

I think that this is considered a new feature, so it should require a minor version update. Can you update the chart version accordingly?
```suggestion
version: 4.6.0
```